density/density.py

```diff
@@ -91,6 +91,39 @@ def authorization_checker(*args, **kwargs):
     return authorization_checker
 
 
+def calculate_fullness_percentage(cur_data):
+    """
+    Calculates percent fullness of all groups and returns the percentage as a
+    list.
+
+    :param list of dictionaries cur_data: data to calculate fullness percentage
+    :return: List of percentage data
+    :rtype: float
+    """
+
+    full_list = []
+
+    with open('data/capacity_group.json') as json_data:
```

Comment:
like anywhere on the root scope. then it will be loaded into memory once when you start the server and not read each time.